setup.py

```diff
@@ -55,7 +55,7 @@
 
     version=version,
 
-    license='Apache License 2.0',
+    license='BSD-3-Clause License',
```

Comment:
The only "BSD-3-Clause" is the SPDX identifier except "License".

```suggestion
    license='BSD-3-Clause',
```